pkg/security/probe/probe_auditing_windows.go

```diff
@@ -0,0 +1,151 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Auditing as a name of the file is too general word. Microsoft terminology used ""security audit"